languages/kurdish/pack/src/main/res/xml/kurdish_dictionaries.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Dictionaries>
+    <Dictionary
+        description="@string/kurdish_dictionary_description"
+        dictionaryResourceId="@array/kurdish_words_dict_array"
+        id="8a678ccc-b62a-41e3-9d1b-0023bf710b2b"
```

Comment:
This `id` files must be unique. Go to https://www.guidgenerator.com/ and generate one to use in your pack.